packages/store/src/execution/internalNgxsExecutionStrategy.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable, Inject } from '@angular/core';
+
+import { NgxsExecutionStrategy, NGXS_EXECUTION_STRATEGY } from './symbols';
+
+@Injectable()
+export class InternalNgxsExecutionStrategy implements NgxsExecutionStrategy {
+  constructor(
+    @Inject(NGXS_EXECUTION_STRATEGY) private _executionStrategy: NgxsExecutionStrategy
+  ) {}
+
+  enter<T>(func: () => T): T {
```

Comment:
rename  `func`  to `tick`

Reply:
Huh? I don't understand why `tick` is a better name?

Reply:
![image](https://user-images.githubusercontent.com/12021443/52411305-66fcc200-2aec-11e9-8655-15c4ff21d53e.png)

better anologue

Reply:
Because in any case, we do it for the angular zones.

Reply:
But this function is doing the work, not the change detection. Looking at the description `tick` definitely doesn't suit this.

Reply:
ok